modules/SofaConstraint/src/SofaConstraint/LCPConstraintSolver.cpp

```diff
@@ -44,6 +45,56 @@ using sofa::core::VecId;
 namespace sofa::component::constraintset
 {
 
+class MechanicalGetConstraintInfoVisitor : public simulation::BaseMechanicalVisitor
```

Comment:
Any reason to set this Visitor directly into the cpp ? I saw it was in the LCPCSolver header before, could not it be in a separate file (along the other mechavisitor) ?
Or is it only useful for LCPSolver and thats why you set in the cpp ("kind of private" class) ?

Reply:
Yes, this visitor is used only in this LCPSolver. I did not like to see the visitor class in the header file, as it hides the main class LCPSolver. I don't mind putting it in a dedicated file if requested.